gradio/cli/commands/components/dev.py

```diff
@@ -63,6 +63,18 @@ def _dev(
         "gradio", gradio_path, cli_arg_name="--gradio-path"
     )
 
+    gradio_node_path = subprocess.Popen(
+        [node, "-e", "console.log(require.resolve('@gradio/preview'))"],
+        cwd=Path(component_directory / "frontend"),
+        stdout=subprocess.PIPE,
```

Comment:
We also need to set `stderror` to Pipe because if the node path is not found, it will be reported to stderror not stdout.